policies/access.md

```diff
@@ -0,0 +1,41 @@
+# GitHub Access Management Policy
+
+## Teams, not Individuals
+
+Access to repositories will be granted only to GitHub teams, not to individuals. A repo may have any number of teams added to it, with the appropriate access levels.
+
+## Team Structure
+
+Teams will be nested, using parent/child relationships, as needed. A child team will implicitly have any privileges its parent team does, so, the most deeply nested team should be the one with the most privileges.
+
+### Top-level teams
+
+Note: this list is intentionally not exhaustive.
+
+- [TAC](https://github.com/orgs/ossf/teams/tac)
+    This team is for current TAC members. Individuals should be added or removed from this team to reflect current membership.
+- [Staff](https://github.com/orgs/ossf/teams/staff)
+    This team is for OpenSSF staff members. Individuals on the top-level team are there as a catch-all, but may be moved to subteams as the need arises.
+  - [PMs](https://github.com/orgs/ossf/teams/pms)
+      This teams is for PMs, who often need Maintain or Admin access on repos.
+  - [Marketing](https://github.com/orgs/ossf/teams/marketing)
+- [Working Groups](https://github.com/orgs/ossf/teams/working-groups)
+    This is the parent team for Working Groups. Every WG should have a subteam contained within this one. All WG subteams must start with `wg-` for consistency.
+- [SIGs](https://github.com/orgs/ossf/teams/sigs)
+    This is the parent team for SIGs. Every SIG should have a subteam contained within this one. All SIG subteams must start with `sig-` for consistency.
+- [Projects](https://github.com/orgs/ossf/teams/projects)
+    This is the parent team for projects. Every project (eg scorecard, AO) should have a subteam contained within this one.
+    Teams for individual repositories go under here, which start with `repo-`, but team names may otherwise be unconstrained.
+
+## GitHub Org Membership
```

Comment:
From https://github.com/ossf/tac/issues/164#issuecomment-1549952618, can we break this section out of this pull request, and give it its own pull request?

We'd like to start the team / role-based access control as soon as possible to help with things like https://github.com/ossf/tac/issues/161 and https://github.com/ossf/tac/issues/162. 

The TAC discussion yesterday on `GitHub Org Membership` was generally in favor, but I think folks would like to ask more clarifying questions and have a longer discussion around this point. But in the meantime, we don't wan to block the team / role-based access control work. Thanks!

Reply:
hmm, that's not how i read @AevaOnline's comment - i read it as needing a separate task to see what could be *built on top of* this section.

The team-based work is blocked on being able to freely add people to the org - and I've been doing that for the last few months as part of my audit already.

Reply:
The way I interpret "membership in the GitHub org should be freely given" is that we would be required to add anyone who asks to https://github.com/ossf, even if they don't end up on any GitHub org teams.

Without this section, my understanding is we would still be able to add people to https://github.com/ossf in order to complete the team-based work.

Reply:
“should” doesn’t obligate us, but yes it’s also fine to have someone in the org on no teams - that makes them easily addable *to* teams.

Fine to remove this section though if the TAC feels it should be deferred.

Reply:
This is done; I'll open the second PR once this one lands.

Reply:
Filed #171